CONTRIBUTING.md

```diff
@@ -83,13 +83,34 @@ Please follow the below conventions when contributing to this project.
 * Each .NET project should have its corresponding test project
 * Each class should have its corresponding test class
 * Each internal and public function in the class should be tested in the unit test class
-* Follow the below naming conventions
+* Follow the below naming conventions:
   * Test Project: *{project name}.(Unit/Functional)Tests* selecting the appropriate type of tests found in the project
   * Test Class: *{class name}Tests.cs*
   * Test methods:
     * (Data)TestMethod: `{method name}_{what is being tested}_{expected outcome}`
     * DataRow (display name): Short description that clearly differentiate between the DataRows
 
+## Code Review Process
+
+### Before Creating a Pull Request
+* The changes cannot break any existing functional/unit tests that are part of the central repository.
+  * This includes all tests, even those not associated with the given feature area.
+* Code submitted must have [basic unit test coverage](#tests), and have all the unit tests pass. Functional tests are encouraged.
+Code submitted must have all the built-in rule tests pass. If applicable, new rules should add [tests for built-in rules](#contributing-analyzer-rules).
+* Code should be commented.
+* Code should be able to merge without any conflicts into the `development` branch.
```

Comment:
Consider rephrasing to something like this.  I think it's also worth moving this point to the top.
``` suggestion
* Code changes should be made in a new branch off the `development` branch.  Pull requests must target `development`.
```